test-ubi.yml

```diff
@@ -0,0 +1,80 @@
+schemaVersion: 2.0.0
```

Comment:
Just noting that these tests are different from test.yml - @guygiat it might be good to note here why UBI doesn't test these cases
```
<   - name: "OpenSSL denies non FIPS compliant algorithms"
<     command: "openssl"
<     args:
<       - md5
<       - /etc/passwd
<     exitCode: 1
<     expectedError: [".*disabled for fips.*"]
<   # holded packages
<   - name: "packages are on hold"
<     command: "apt-mark"
<     args:
<       - "showhold"
<     expectedOutput:
<       - ".*ca-certificates.*"
<       - ".*libssl1.0.0.*"
<       - ".*libssl1.1.*"
<       - ".*openssl.*"
<   - name: "Ruby denies FIPS compliant algorithms"
<     command: "ruby"
<     args:
<       - -ropenssl
<       - -e
<       - 'OpenSSL.fips_mode = true'
<       - -ea
<       - 'puts(OpenSSL::Digest::MD5.new)'
<     exitCode: 1
<     expectedError: [".*disabled for fips.*"]
```

Reply:
- The openssl md5 on UBI is supported because FIPS needs be enforced on the host level in order to disable md5 on openssl.
-  On the ubuntu base image we hold the openssl, libssl, ca-certificates packages so we can test it's actually being hold. For UBI I didn't do it because we install those packages from the ubi repo.

Reply:
Please add the OPENSSL_FIPS=1 environment variable, this will enable FIPS mode for an openssl command.

Reply:
@guygiat I'm still waiting on the dockerhub project - can you make this change that @aloncarmel111 requested before we merge? ^

Reply:
@aloncarmel111 When I set the OPENSSL_FIPS=1 in ubi I'm getting `FIPS mode not supported.` when trying to run openssl commands, I think this environment variable is only supported on ubuntu/other flavours than redhat.
Are you familiar with this?

Reply:
@guygiat Seems that to enable container FIPS mode the host should also set to in FIPS mode see [here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/security_guide/chap-federal_standards_and_regulations#sec-Enabling-FIPS-Mode).
The ruby flag `fips_mode` should works and ruby should deny MD5 algorithm can you add this verification please.

Reply:
Sure no problem.
@izgeri To summarise: I've added the ruby md5 check so we'll drop only the openssl md5 test and the package hold test.